backend/app/auth/jwt.py

```
@@ -0,0 +1,24 @@
+
+def decode_access_token(token: str):
+    try:
+        payload = jwt.decode(
+            token,
+            settings.JWT_SECRET,
+            algorithms=[settings.JWT_ALGORITHM]
+        )
+        print("Decoded JWT:", payload)   
+        if "sub" in payload:
+            payload["sub"] = int(payload["sub"])
+        return payload
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)